Core/Config.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2012- PPSSPP Project.
+﻿// Copyright (c) 2012- PPSSPP Project.
```

Comment:
Hmm, supposedly 2017 uses editorconfig by default, but I haven't tried it yet:

https://blogs.msdn.microsoft.com/dotnet/2016/12/15/code-style-configuration-in-the-vs2017-rc-update/

-[Unknown]